docs/source/common_issues.rst

```diff
@@ -505,11 +505,19 @@ to see the types of all local variables at once. Example:
    #     b: builtins.str
 .. note::
 
-   ``reveal_type`` and ``reveal_locals`` are only understood by mypy and
-   don't exist in Python. If you try to run your program, you'll have to
-   remove any ``reveal_type`` and ``reveal_locals`` calls before you can
-   run your code. Both are always available and you don't need to import
-   them.
+   ``reveal_type`` and ``reveal_locals`` are understood by mypy during typechecking,
+   and don't have to be imported as functions. However,
+   if you don't import them, then they don't exist at runtime! Therefore,
+   you'll have to remove any ``reveal_type`` and ``reveal_locals`` calls from your program
+   or else Python will give you an error at runtime about those names being undefined.
+   Alternatively, you can import ``reveal_type`` from ``typing_extensions``
+   (or, in more recent versions of python, from ``typing``)
```

Comment:
Does intersphinx work here? Could we link to `typing.reveal_type`?

Reply:
I don't know why, but this doesn't seem to work for me. Even though the intersphinices to [sys.version_info](https://docs.python.org/3/library/sys.html#sys.version_info) and typing.TYPE_CHECKING seem to work, typing.reveal_type won't generate for me. Maybe I'm doing something wrong locally and there's some caching problem I'm running into. Anyway, it's harmless to include an intersphinx to typing.reveal_type here because it gracefully degrades down to `normal code`, so I will.

Reply:
Ah, nevermind: I actually get a hard error when I try to push that: `py:data reference target not found: typing.reveal_type`

Reply:
Unfortunately I can't mark this as resolved :(

Reply:
I'm not sure if you mean "can't" as in "according to your own principles" or "the UI doesn't let you". I'm assuming the latter, and the UI does let me, so I've done that 🙂.